### ISSUE TITLE
A MessageBox extension that allows "Further Details" to be shown

### ISSUE THREAD

**Issue by Smurf-IV** [Wagnerp/Krypton-Toolkit-Suite-Extended-NET-4.7#7]
Press a button, and a slide down panel will be shown with "Application" extended more info style..
Could be an RTF control.
Will still need to Ctrl+C compatible.

**Comment by Wagnerp** [comment]
https://github.com/Nexus-Mods/Nexus-Mod-Manager/blob/master/UI/Controls/ExtendedMessageBox.cs

**Comment by Smurf-IV** [comment]
Not allow4ed to use that code due to "GNU General Public License v2.0"

**Comment by Wagnerp** [comment]
Ahh ok, I'm trying to create the message boxes using winforms. Message boxes with 'extra details' may require new controls to be written. I have been trying to find some controls to use, but haven't found any yet. Also, how big should the message box icon be (in pixels), as I intend users to have the option to set a custom one alongside default ones, any thoughts?

**Comment by Smurf-IV** [comment]
Have a look at the KryptonMessageBox code as the start point.
Also the WPF Message has prettier (relative) and slightly larger icons.

**Comment by Wagnerp** [comment]
Yes I have extracted the code as a reference point. I'm going to look into how best to do a timeout function, a progress bar might be a nice feature here, possibly attached to a bool argument to toggle viability of it?

**Comment by Smurf-IV** [comment]
Most of these messageBox enhancements could be covered as a subset of a "Task Dialog"
https://www.developerfusion.com/article/71793/windows-7-task-dialogs/
I think once the CommandLink button is sorted then the above would be able to do the rest of the MessageBox things.

**Comment by Wagnerp** [comment]
I do have a Windows API Code Pack Nuget package already available if this helps.

**Comment by Smurf-IV** [comment]
I have a `CommandLink C# Button` already in my projects "https://github.com/Smurf-IV/Elucidate/tree/master/Elucidate/Elucidate/Shared"
It just need to be put into "Krypton Language" so that the layout is corrected. 
You can assign that request to me if you like.

**Comment by Wagnerp** [comment]
https://github.com/JohannBlais/InformationBox
